capstone/src/main/java/com/google/univiz/collegescorecarddata/CollegeScoreCard.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.collegescorecarddata;
+
+import java.util.List;
+
+public interface CollegeScoreCard {
+  /** General Use Functions */
+  boolean validateCollegeName(String collegeName);
+
+  List<UniversityData> getCollegesData(List<String> collegeNames);
+
+  /** Search Feature Functions */
+  List<UniversityData> getCollegeSuggestions(String currName);
```

Comment:
should this be in the SearchData interface? actually looks like methods you refactored into other interfaces need to get removed from this interface...

Reply:
yes-fixed that!